.travis.yml

```diff
@@ -69,52 +61,6 @@ matrix:
         - MATRIX_EVAL="CXX=clang++-6.0 && CXXFLAGS_OS=-stdlib=libc++"
           TESTFOLDER="src/test/unit/services src/test/unit/variational src/test/unit/version_test.cpp"
           PARALLEL=2
-    - <<: *linux_gcc
```

Comment:
No need to duplicate these tests anymore on Travis. I left clang ubuntu tests which should stay in my opinion. 

Linux is the only platform where two different compilers are used commonly. On windows clang++ does not currently work with Stan, and g++ on macOS is very rare.

We should maybe switch to 18.04, this is still using 14.04. But that is a separate issue.